lib/handle-pull-request.ts

```diff
@@ -0,0 +1,104 @@
+import * as core from "@actions/core";
+import * as github from "@actions/github";
+import { readFileSync } from "fs";
+import localeDate from "./locale-date";
+import type {
+  PullRequestEvent,
+  SimplePullRequest,
+} from "@octokit/webhooks-types";
+import {
+  getScheduleDateString,
+  hasScheduleCommand,
+  isFork,
+  isValidDate,
+  stringifyDate,
+} from "./utils";
+import {
+  createComment,
+  deleteComment,
+  generateBody,
+  getPreviousComment,
+  updateComment,
+} from "./comment";
+
+/**
+ * Handle "pull_request" event
+ */
+export default async function handlePullRequest(): Promise<void> {
+  if (!process.env.GITHUB_TOKEN) {
+    core.setFailed("GITHUB_TOKEN environment variable is not set");
+    return;
```

Comment:
No need to use `process.exit(1)` like before since `core.setFailed` already does that.